scripts/packergen.py

```diff
@@ -49,8 +49,13 @@
 
 echo "--> Cleaning home directories..."
 for user in $(cat /var/lib/google/google_users); do
-  pkill -u $user sshd || echo "  > $user: NO PROCESS TO KILL..."
+  echo "---> Checking user $user ..."
+  pkill -u $user || echo "  > $user: NO PROCESS TO KILL..."
   userdel -r $user && echo "  > $user: REMOVED"
+  if [[ "$?" -ne 0 ]]; then
```

Comment:
I think it's nice to log this, but wasn't it work the same before? Please note that we define `-eu` flag here.

Reply:
Hey @wgrzelak !

Makes sense regarding the `-eu`. 

Regarding `userdel`, no. I've removed the `sshd` pattern in order to purge all the processes.

So, as per chat conversation, will remove the log part.